### PR TITLE
KafkaBetaRelease+AddConnectorNameInMetadata

### PR DESCRIPTION
**Pull request by xinlian12**
- Kafka beta 1.0.0 release
- Add connector name in the metadata
```
{
    "id": "SampleDatabase_7QdKALblBic=_cosmosdb-source-connector2-3",
    "metadata": {
        "feedRanges": "[\"eyJSYW5nZSI6eyJtaW4iOiIiLCJtYXgiOiIwNUMxQzlDRDY3MzM5OCJ9fQ==\",\"eyJSYW5nZSI6eyJtaW4iOiIwNUMxQzlDRDY3MzM5OCIsIm1heCI6IjA1QzFEOUNENjczMzk4In19\",\"eyJSYW5nZSI6eyJtaW4iOiIwNUMxRDlDRDY3MzM5OCIsIm1heCI6IjA1QzFFMzk5Q0Q2NzMyIn19\",\"eyJSYW5nZSI6eyJtaW4iOiIwNUMxRTM5OUNENjczMiIsIm1heCI6IjA1QzFFOUNENjczMzk4In19\",\"eyJSYW5nZSI6eyJtaW4iOiIwNUMxRTlDRDY3MzM5OCIsIm1heCI6IkZGIn19\"]"
    }
}
```

### CODE REVIEW

File: sdk/cosmos/azure-cosmos-kafka-connect/src/main/java/com/azure/cosmos/kafka/connect/CosmosSourceConnector.java [hunk 1]
```diff
@@ -55,11 +56,14 @@
  */
 public final class CosmosSourceConnector extends SourceConnector implements AutoCloseable {
     private static final Logger LOGGER = LoggerFactory.getLogger(CosmosSourceConnector.class);
+    private static final String CONNECTOR_NAME = "name";
```
Comment (kushagraThapar) [inline sdk/cosmos/azure-cosmos-kafka-connect/src/main/java/com/azure/cosmos/kafka/connect/CosmosSourceConnector.java:59]: what is the default value here? Just the string `name`?

File: sdk/cosmos/azure-cosmos-kafka-connect/src/main/java/com/azure/cosmos/kafka/connect/implementation/source/ContainersMetadataTopicPartition.java [hunk 2]
```diff
@@ -12,22 +12,30 @@
...
 
     private final String databaseName;
+    private final String connectorName;
 
-    public ContainersMetadataTopicPartition(String databaseName) {
+    public ContainersMetadataTopicPartition(String databaseName, String connectorName) {
         checkArgument(StringUtils.isNotEmpty(databaseName), "Argument 'databaseName' can not be null");
 
         this.databaseName = databaseName;
+        this.connectorName = StringUtils.isEmpty(connectorName) ? StringUtils.EMPTY : connectorName;
```
Comment (kushagraThapar) [inline sdk/cosmos/azure-cosmos-kafka-connect/src/main/java/com/azure/cosmos/kafka/connect/implementation/source/ContainersMetadataTopicPartition.java:24]: Shouldn't we have a good default value?

**Review by kushagraThapar — APPROVED** [review]

File: sdk/cosmos/azure-cosmos-kafka-connect/src/main/java/com/azure/cosmos/kafka/connect/CosmosSourceConnector.java (hunk 1 above)
Comment (xinlian12) [reply sdk/cosmos/azure-cosmos-kafka-connect/src/main/java/com/azure/cosmos/kafka/connect/CosmosSourceConnector.java:59]: the connector name should actually be forced from kafka side, during my testing, if I do not provide a name, an error will be thrown from kafka side, so supposedly we should not never get an empty name

File: sdk/cosmos/azure-cosmos-kafka-connect/src/main/java/com/azure/cosmos/kafka/connect/implementation/source/ContainersMetadataTopicPartition.java (hunk 2 above)
Comment (xinlian12) [reply sdk/cosmos/azure-cosmos-kafka-connect/src/main/java/com/azure/cosmos/kafka/connect/implementation/source/ContainersMetadataTopicPartition.java:24]: want to keep this way for now to show that there is no connector name being configured.

**Pull request merged**